pkgs/applications/networking/pjsip/default.nix

```diff
@@ -41,21 +45,28 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ openssl libsamplerate ]
     ++ lib.optional stdenv.isLinux alsa-lib
-    ++ lib.optional stdenv.isDarwin AppKit;
+    ++ lib.optionals stdenv.isDarwin [ AppKit CoreFoundation Security ];
 
-  preConfigure = ''
-    export LD=$CC
-  '';
+
```

Comment:
```suggestion
```

Reply:
Removing this is unfortunately not possible. I've only moved it. When I tried to remove this I got:

```
       > /nix/store/fhzz4yrdy17czwc9i4swhlpcp445inzb-binutils-2.40/bin/ld: unrecognized option '-Wl,-soname,libpj.so.2'
       > /nix/store/fhzz4yrdy17czwc9i4swhlpcp445inzb-binutils-2.40/bin/ld: use the --help option for usage information
```